sdk/python/packages/flet-core/src/flet_core/textfield.py

```diff
@@ -54,18 +53,22 @@ class TextCapitalization(Enum):
 @dataclasses.dataclass
 class InputFilter:
     regex_string: str
-    allow: bool = field(default=True)
-    replacement_string: str = field(default="")
+    allow: bool = True
+    replacement_string: str = ""
+    multiline: bool = False
+    case_sensitive: bool = True
+    unicode: bool = False
+    dot_all: bool = False
```

Comment:
wouldn't be more "handy" to have `unicode = True` and `case_sensitive = False` by default?

Reply:
Those are actually the default flutter values of `RegExp`.
What's your idea?

Reply:
Apparently, unicode mode is not what I thought: https://api.flutter.dev/flutter/dart-core/RegExp/isUnicode.html
OK, let's keep defaults as they are in RegExp.